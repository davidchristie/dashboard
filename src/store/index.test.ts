import { Store } from "redux";

import { createReduxStore } from ".";

describe("redux store", () => {
  let store: Store;

  beforeEach(() => {
    store = createReduxStore();
  });

  describe("initial state", () => {
    it("matches snapshot", () => {
      expect(store.getState()).toMatchSnapshot();
    });
  });
});
