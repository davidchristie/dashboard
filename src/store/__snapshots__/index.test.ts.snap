// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`redux store initial state matches snapshot 1`] = `
Object {
  "order": Object {
    "details": Object {},
    "list": Array [],
  },
  "product": Object {
    "details": Object {},
    "list": Array [],
  },
  "router": Object {
    "action": "POP",
    "location": Object {
      "hash": "",
      "pathname": "/",
      "query": Object {},
      "search": "",
      "state": undefined,
    },
  },
}
`;
